solr/solr-ref-guide/modules/getting-started/pages/tutorial-diy.adoc

```diff
@@ -53,6 +53,9 @@ Local Files with `bin/solr post`::
 If you have a local directory of files, the Post Tool (`bin/solr post`) can index a directory of files.
 We saw this in action in our first exercise.
 +
+// NOCOMMIT: The user will need to add /update/extract handler?
+// TODO: The user will need to start a Tika server
```

Comment:
@epugh Did this tutorial ever work? It uses the `_default` configset but does not add an `/update/extract/` handler before POST'ing docs.